packages/react-native-editor/__device-tests__/helpers/caps.js

```diff
@@ -13,6 +13,7 @@ const ios = {
 
 exports.iosLocal = {
 	...ios,
+	waitForQuiescence: true,
```

Comment:
Here is the documentation about this capability:
`It allows to turn on/off waiting for application quiescence in WebDriverAgent, while performing queries. The default value is true. You can avoid this kind of issues if you turn it off. Consider using waitForIdleTimeout capability instead for this purpose since Appium 1.20.0`

Looks like that by default is `true` but I noticed that explicitly setting it fixed the problem of booting up the iOS simulator 🤷‍♂️ .